met-web/src/components/common/Table/index.tsx

```diff
@@ -204,9 +192,9 @@ function MetTable<T>({
                 <TablePagination
                     rowsPerPageOptions={[5, 10, 25]}
                     component="div"
-                    count={filteredRows.length}
-                    rowsPerPage={rowsPerPage}
-                    page={page}
+                    count={Number(total)}
```

Comment:
not sure why you had to cast the value to number here, in the def it's already a number, can you remove the cast?

Reply:
also is it total pages or total records?

Reply:
good eye, you're right unnecessary. And it is total records in DB